blessedDeps.gradle

```diff
@@ -68,5 +69,8 @@ rootProject.ext.deps = [
     lithoWidget           : "com.facebook.litho:litho-widget:$LITHO_VERSION",
     lithoAnnotations      : "com.facebook.litho:litho-annotations:$LITHO_VERSION",
     paris                 : "com.airbnb.android:paris:$PARIS_VERSION",
-    parisProcessor        : "com.airbnb.android:paris-processor:$PARIS_VERSION"
+    parisProcessor        : "com.airbnb.android:paris-processor:$PARIS_VERSION",
+    glide                 : "com.github.bumptech.glide:glide:$GLIDE_VERSION",
+    glideProcessor        : "com.github.bumptech.glide:compiler:$GLIDE_VERSION",
+    glideRecyclerView     : "com.github.bumptech.glide:recyclerview-integration:$GLIDE_VERSION"
```

Comment:
I don't think `glideRecyclerView` is necessary, the epoxy preloader code shouldn't depend on it